client/ladder/ladder.tsx

```diff
@@ -152,6 +153,7 @@ export interface LadderTableProps {
 
 export function LadderTable(props: LadderTableProps) {
   const [dimensionsRef, containerRect] = useDimensions()
+  const nowRef = useRef(Date.now())
```

Comment:
This value needs to be updated when the data is refreshed, might be easier to just use a memoized value with a dependency on the ladder data or something (we don't really care too much if it gets updated unnecessarily occasionally, just as long as it isn't getting updated every render)